### ISSUE TITLE
This game does not require use of the word 'verb' is misleading

### ISSUE THREAD

**Issue by warrigal24** [ainslec/adventuron-issue-tracker#264]
I have struck a few situations where I've used a verb and the response says, "This game does not require use of the word 'verb'." only to find that somewhere else in the game, I actually DO require that verb. I think the default response occurs if the author has not provided a default handler for that verb. In this case, it would be better to say something like "I can't do that now." or "I don't understand." rather than to give the false impression that the verb is never required.

**Comment by ainslec** [comment]
Specific example please.

**Comment by warrigal24** [comment]
Unfortunately, I didn't record any specific examples. It might have been in situations where the author used the verb as a wild card. I guess there's no control over this. For example:
```
   : match "_ me" {
      : print "You can't {verb_phrase} yourself.";
      : done;
   }
```
Sample transcript:
```
>LOVE GIRLFRIEND
This game does not require use of the word 'love'.

>LOVE ME
You can't love yourself.
```

**Comment by ainslec** [comment]
Needs some thought.

Actually this message is one of the most useful messages in the whole engine, and I certainly would not replace it with something less useful.

Prior to introducing the This game does not require use of the word 'verb'. messages, it was really hart to know if verbs were understood or not understood in a game. I watched a playthough where a player constantly used the wrong verbs without trying other verbs. It's a service to the player to tell them which verbs are known by the game and which are not, although in the case of wildcard verbs, it may be that the message needs to be reworked to reflect ambiguity.

**Comment by warrigal24** [comment]
Agreed. If you get this message, then you should be able to make a mental note not to bother using this verb again.

I must have just been unlucky or something, but I could have sworn that I've struck this quite a few times, including in my own game. As a consequence, I am now very careful to always provide a default response for all verbs.
